app/src/components/PreviewEvent/PreviewEventContainer.tsx

```diff
@@ -21,8 +21,8 @@ export const PreviewEventContainer = () => {
     return <div>Det finnes ingen event å forhåndsvise</div>;
   }
 
-  const participantsText = `0 av ${event.maxParticipants ?? '∞'}${
-    event.hasWaitingList ? ' og 0 på venteliste' : ''
+  const participantsText = `0 av ${event.maxParticipants === 0 ? '∞' : event.maxParticipants}${
+    event.hasWaitingList && event.maxParticipants != 0 ? ' og 0 på venteliste' : ''
```

Comment:
Føler ikkje `hasWaitingList` burde kunne vere true dersom man har `maxParticipants == 0`? 🤔

Reply:
Nei er enig i at det ikke burde være mulig, men slik løsningen er nå kan man krysse av for at man har venteliste og ubegrensede plasser. Så denne ble lagt inn for å sikre at det ikke står "x av ∞ påmeldte og 0 på venteliste" slik det gjorde nå om noen krysser av venteliste. 

Vi kan evt skjule muligheten for å ha venteliste frem til man setter en begrensning på arrangementet for å slippe dette.